documentation/src/main/asciidoc/topics/con_jgroups_encrypt.adoc

```diff
@@ -0,0 +1,49 @@
+[id='jgroups_encryption-{context}']
+= {brandname} Cluster Security
+To secure cluster traffic, you configure {brandname} nodes to encrypt JGroups message payloads with secret keys.
+
+{brandname} nodes can obtain secret keys from either:
+
+* The coordinator node (asymmetric encryption).
+* A shared keystore (symmetric encryption).
+
+.Retrieving secret keys from coordinator nodes
+
+You configure asymmetric encryption by adding the `ASYM_ENCRYPT` protocol to a JGroups stack in your {brandname} configuration.
+This allows {brandname} clusters to generate and distribute secret keys.
+
+[IMPORTANT]
+====
+When using asymmetric encryption, you should also provide keystores so that nodes can perform certificate authentication and securely exchange secret keys.
+This protects your cluster from man-in-the-middle (MitM) attacks.
+====
+
+Asymmetric encryption secures cluster traffic as follows:
+
+. The first node in the {brandname} cluster, the coordinator node, generates a secret key.
+. A joining node performs certificate authentication with the coordinator to mutually verify identity.
+. The joining node requests the secret key from the coordinator node. That request includes the public key for the joining node.
+. The coordinator node encrypts the secret key with the public key and returns it to the joining node.
```

Comment:
is the message signed? what if an attacker injects a message with some other key encrypted (basically a DoS attack)?

//crap, I'm overthinking again...